carto/import_api.py

```diff
@@ -113,6 +116,25 @@ def __init__(self, url, client, api_version='v1', interval=None, **kwargs):
         super(URLImport, self).__init__(client, **kwargs)
 
 
+class CartoExportJob(ImportJob):
+    def __init__(self, client, visualization_id, api_key, api_version='v1', **kwargs):
+        self.viz_id = visualization_id
+        self.api_key = client.api_key
+
+        super(CartoExportJob, self).__init__(client, **kwargs)
+
+    def run(self, **import_params):
+        import_params["visualization_id"] = self.viz_id
+        self.send(EXPORT_API_URL, url_params=import_params, client_params={"http_method": "POST"})
```

Comment:
in order to make the api version configurable, you also need to have an internal attribute api_url, and not use EXPORT_API_URL directly (you can take a look at sql_api.py, for a similar approach)